code/modules/vending/clothesmate.dm

```diff
@@ -7,7 +7,8 @@
 	product_slogans = "Dress for success!;Prepare to look swagalicious!;Look at all this swag!;Why leave style up to fate? Use the ClothesMate!"
 	vend_reply = "Thank you for using the ClothesMate!"
 	light_color = LIGHT_COLOR_GREEN
-	products = list(/obj/item/clothing/head/beanie = 3,
+	products = list(/obj/item/comb = 10,
```

Comment:
You have a bunch of clothing and then ten combs? you should just place a couple in bathrooms/barbershop instead